### PR TITLE
Car Price Prediction using DL #339

### PR DESCRIPTION
**Pull request by Soumiksb06**
# Pull Request for DL-Simplified 💡

## Issue Title : Car Price Prediction using DL #339

- **Info about the related issue (Aim of the project)** : The goal of the project is to predict car prices using deep learning models. The dataset contains various features, both categorical and numerical, and different DL models are implemented for the task. The best model is chosen based on accuracy scores. 
- **Name:** Soumik Banerjee
- **GitHub ID:** https://github.com/Soumiksb06/
- **Email ID:**  soumikbanerjee230@gmail.com 
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)**  SWOC 2024 contributor.

Closes: #339

### Describe the add-ons or changes you've made 📃

I have added a detailed README file to the project, providing information on installation, usage, visualizations, models, results, and conclusions. Additionally, I have included visualizations such as pair plots, correlation heatmap, box plots, and an interactive 3D scatter plot. The README also presents the DL models used, their architectures, and compilation details.

## Type of change ☑️

What sort of change have you made:

- [x] My code follows the code style of this project.
- [x] Developed different DL models.
- [x] Chose the best model.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
>
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

I have tested the code by running it and verifying that the visualizations are generated correctly. I also ensured that the DL models are trained and evaluated properly.

## Checklist: ☑️

Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Put the `.ipynb` file inside the Model folder.
2. In the `README.md` file, update the Conclusion section by adding a table of accuracy scores instead of comma separated way. You can also add the visualization output of the accuracy scores there, that will make the README more attractive.
3. Overall it looks really good, need to fix these minor changes.

@Soumiksb06

**Comment by Soumiksb06** [comment]
> 1. Put the `.ipynb` file inside the Model folder.
> 2. In the `README.md` file, update the Conclusion section by adding a table of accuracy scores instead of comma separated way. You can also add the visualization output of the accuracy scores there, that will make the README more attractive.
> 3. Overall it looks really good, need to fix these minor changes.
> 
> @Soumiksb06 

1. Ok
2. I think I already included visualization output of accuracy scores there.
3. Okk, thanks.

File: Car Price Prediction using DL/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+![Heatmap](../Images/CorrelationHeatmap.png)
+
+Box plot for car body vs. price
+![Box plot](../Images/box_CarBody_vs_Price.png)
+
+Interactive 3D scatter plot for curb weight, horsepower, and price
+![Interactive Plot](../Images/interactive_3d_scatter_plot.svg)
+
+Error Values Comparison Bar Plot
+![Error Values Plot](../Images/ErrorValuesBarPlot.png)
```
Comment (abhisheks008) [inline Car Price Prediction using DL/Model/README.md:43]: Shift this image in the Conclusion section along with the table which I requested you to update.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Car Price Prediction using DL/Model/README.md (hunk 1 above)
Comment (Soumiksb06) [reply Car Price Prediction using DL/Model/README.md:43]: > Shift this image in the Conclusion section along with the table which I requested you to update.

Ok ok, the error values plot na?

Comment (Soumiksb06) [reply Car Price Prediction using DL/Model/README.md]: > Shift this image in the Conclusion section along with the table which I requested you to update.

Modified as you said. Please check once now.

**Review by abhisheks008 — APPROVED** [review]
Fantastic work done. Your PR is approved and ready to be merged.
@Soumiksb06

**Pull request merged**